userguide/object_detection/export-coreml.md

```diff
@@ -37,6 +37,8 @@ For more details on the integration with Core ML and a sample app to get
 you started, please look at the the article on
 [Recognizing Objects in Live Capture](https://developer.apple.com/documentation/vision/recognizing_objects_in_live_capture).
 
+**Note:** The bounding box object `VNRecognizedObjectObservation.boundingBox` has a different definition from the one used for Turi Create. First, the location of the bounding box is defined by the upper-left corner of the rectangle instead of the center. Secondly, the location, width, and height for the bounding box are all normalized between [0,1] by the dimensions of the images.
```

Comment:
I believe Vision uses the lower left corner as the origin of the coordinates, and the lower-left corner of the bounding box to define the position. Is this not what you're observing?

https://developer.apple.com/documentation/vision/vndetectedobjectobservation/2867227-boundingbox?language=objc

Reply:
Just changed the documentation.